### PR TITLE
Add Sup(Stack Up)

### PR DESCRIPTION
**Pull request by kureikain**
This add https://github.com/pressly/sup
Stack Up is a simple deployment tool that performs given set of commands
on multiple hosts in parallel. It reads Supfile, a YAML configuration
file, which defines networks (groups of hosts), commands and targets.

### CODE REVIEW

File: Library/Formula/sup.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+require "language/go"
+
+class Sup < Formula
+  desc "Super simple deployment tool - just Unix - think of it like 'make' for a network of servers."
```
Comment (bfontaine) [inline Library/Formula/sup.rb]: Please shorten this. `sup: <description>` should fit under 80 characters.

File: Library/Formula/sup.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  homepage "https://github.com/pressly/sup"
+  url "https://github.com/pressly/sup/archive/4ee5083c8321340bc2a6410f24d8a760f7ad3847.zip"
```
Comment (bfontaine) [inline Library/Formula/sup.rb]: We need a stable release here.

File: Library/Formula/sup.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+    mkdir_p buildpath/"src/github.com/pressly/"
+    ln_sf buildpath, buildpath/"src/github.com/pressly/sup"
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    system "go", "build", "-o", bin/"sup", "./cmd/sup"
+  end
+
+  test do
+    assert_equal "0.3", shell_output("#{bin}/bin/sup")
```
Comment (bfontaine) [inline Library/Formula/sup.rb]: Can’t we write a test that does something more substantial than e.g. printing the version?

File: Library/Formula/sup.rb [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  end
+
+  go_resource "gopkg.in/yaml.v2" do
+    url "https://gopkg.in/yaml.v2.git", :revision => "f7716cbe52baa25d2e9b0d0da546fcf909fc16b4"
+  end
+
+  def install
+    ENV["GOBIN"] = bin
+    ENV["GOPATH"] = buildpath
+    ENV["GOHOME"] = buildpath
```
Comment (bfontaine) [inline Library/Formula/sup.rb]: Why is setting `GOHOME` necessary? Did you mean `GOROOT`?

**Comment by bfontaine** [comment]
Ping?

**Comment by DomT4** [comment]
Closing as dead but will take a resubmission with the comments addressed if you're still interested. Thank you nonetheless for the submission to Homebrew :heart:

**Pull request closed without merging**